File: src/classes/api/req_res/get-ur-logs-response.ts
// ˅
'use strict';

import { URLog } from '@/classes/datas/ur-log';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class GetURLogsResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    urlogs: Array<URLog>;

    constructor() {
        // ˅
        super()
        this.urlogs = new Array<URLog>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/app/gkill/api/req_res/get_ur_logs_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type GetURLogsResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	URLogs []*reps.URLog `json:"urlogs"`
}
